src/Response/DTO/ParcelShop.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\OverseasBundle\Response\DTO;
+
+class ParcelShop
+{
+    private int $countryId;
+    private int $centerId;
+    private string $shortName;
+    private string $remark = '';
+    private ?bool $delivery;
+    private ?bool $dropoff;
+    private ?float $geoLat;
+    private ?float $geoLong;
```

Comment:
Same here for a Coordinates VO

Reply:
Also public properties instead public methods

Reply:
Check out my solutions for this on this PR https://github.com/answear/overseas-bundle/pull/6